src/app/test/page.tsx

```
@@ -0,0 +1,31 @@
+"use client";
+
+import { parseCampersCSV } from "@/data/storage/parseCampersCSV";
+import { useRef } from "react";
+
+export default function TestPage() {
+  let fileInpRef = useRef<HTMLInputElement | null>(null);
```

ESLint

[error] 'fileInpRef' is never reassigned. Use 'const' instead.

(prefer-const)